frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -305,9 +306,49 @@ const SECTIONS = updateSectionsWithPlugins({
       },
     ],
   },
+  uploads: {
+    name: t`Uploads`,
+    order: 85,
+    adminOnly: true,
+    settings: [
+      {
+        key: "uploads-enabled",
+        display_name: t`Data Uploads`,
+        description: t`Enable admins to upload data to new database tables from CSV files.`,
+        type: "boolean",
+      },
+      {
+        key: "uploads-database-id",
+        getHidden: settings => !settings["uploads-enabled"],
+        display_name: t`Database`,
+        description: t`Identify a database where upload tables will be created. Only actions-enabled databases can be used for data uploads.`,
+        placeholder: t`Select a database`,
+        widget: UploadDbWidget,
+      },
+      {
+        key: "uploads-schema-name",
+        getHidden: settings =>
```

Comment:
This should only be visible if the database supports schema. SQLite does not for example

Reply:
yeah this is weirdly hard with our settings. I was gonna follow up on that nuance rather than blocking it for that alone.

Reply:
Okay. It could also be delayed to v2 since all the main DBs (H2, MySQL, Postgres) have schemas